### DISCUSSION TITLE
Unable to group Terraform and Docker packages with same name

### DISCUSSION THREAD

**Discussion by notclive** [renovatebot/renovate#9563] [Request Help] [answered]
I'm trying to update the version of the Docker image that runs my Terraform scripts, with the Terraform version specified in the scripts, but none of the configurations I've tried has worked.

My packageRule is
`{
    "matchPackageNames": ["hashicorp/terraform"],
    "groupName": "terraform"
},`

I have other package rules that are grouping packages together, so I don't think it's a wider problem with my config.

These are the two MR's I see, they both show the package name as `hashicorp/terraform`.

![image](https://user-images.githubusercontent.com/964928/114832526-7bbcfa80-9dc6-11eb-9eb3-e963389cd9ad.png)
![image](https://user-images.githubusercontent.com/964928/114832569-85def900-9dc6-11eb-9de1-db6022219198.png)

**Comment by viceice** [comment, marked as answer]
It's because renovate adds an additional branch prefix by default, do two different branches are producing two pr's.

Overide `additionalBranchPrefix` in your rule to an empty string.

This behavior will change soon with upcoming renovate v25.

**Reply by rarkins** [reply]
i.e. `{ "matchPackageNames": ["hashicorp/terraform"], "groupName": "terraform", "additionalBranchPrefix": "" }`

**Reply by notclive** [reply]
Thank you, that worked perfectly
